app/core/store/modules/tracker.js

```diff
@@ -1,9 +1,19 @@
+const DEFAULT_TRACKING_DOMAINS = [
+  'codecombat.com'
+];
+
+const COCO_ENABLE_TRACKING_OVERRIDE_QUERY_PARAM = 'coco_tracking'
+
 export default {
   namespaced: true,
 
   state: {
     doNotTrack: window.navigator && window.navigator.doNotTrack === "1",
     spying: window.serverSession && typeof window.serverSession.amActually !== 'undefined',
+    trackingEnabledForEnvironment: DEFAULT_TRACKING_DOMAINS.includes(window.location.hostname),
```

Comment:
Expected to track direct.codecombat.com?

Reply:
Can you clarify?  Are you saying that we should track but are not?  Or we shouldn't track but are?

Reply:
Merging but will follow up with another PR if we need to!